homework/hw01-tolstoy.py

```diff
@@ -0,0 +1,16 @@
+from pyspark import SparkContext
```

Comment:
pyspark definitely overkill here, and certainly won't just run on folks' computers